api/message.go

```diff
@@ -161,15 +162,35 @@ type EditMessageData struct {
 	Flags *discord.MessageFlags `json:"flags,omitempty"`
 }
 
-// EditMessage edits a previously sent message. For more documentation, erfer to
+// EditText edits the contents of a previously sent message. For more
+// documentation, refer to EditMessageComplex.
+func (c *Client) EditText(
+	channelID, messageID discord.Snowflake, content string) (*discord.Message, error) {
+
+	return c.EditMessageComplex(channelID, messageID, EditMessageData{
+		Content: option.NewNullableString(content),
+	})
+}
+
+// EditEmbed edits the embed of a previously sent message. For more
+// documentation, refer to EditMessageComplex.
+func (c *Client) EditEmbed(
+	channelID, messageID discord.Snowflake, embed discord.Embed) (*discord.Message, error) {
+
+	return c.EditMessageComplex(channelID, messageID, EditMessageData{
+		Embed: &embed,
+	})
+}
+
+// EditMessage edits a previously sent message. For more documentation, refer to
 // EditMessageComplex.
 func (c *Client) EditMessage(
 	channelID, messageID discord.Snowflake, content string,
-	embed *discord.Embed, suppressEmbeds bool) (*discord.Message, error) {
+	embed discord.Embed, suppressEmbeds bool) (*discord.Message, error) {
```

Comment:
Can this be kept as a pointer? This will break quite a few APIs, and making it a pointer allows for optional.

Reply:
Of course, I didn't think that one through.